efel/io.py

```diff
@@ -103,6 +103,7 @@ def load_neo_file (file_name, stim_start=None, stim_end=None):
             returned object : [Segments_1, Segments_2, ..., Segments_n] 
             Segments_1 = [Traces_1, Traces_2, ..., Traces_n] 
     """
+
```

Comment:
the load_neo_file is quite long, and factoring out the detection of start and end points would probably make it more clear.  That might also be useful code to have within the neo library, as it seems like common functionality that others would want (potentially?).